tests/instances.tcl

```diff
@@ -102,6 +102,7 @@ proc spawn_instance {type base_port count {conf {}} {base_conf_file ""}} {
             puts $cfg [format "tls-client-key-file %s/../../tls/client.key" [pwd]]
             puts $cfg [format "tls-dh-params-file %s/../../tls/redis.dh" [pwd]]
             puts $cfg [format "tls-ca-cert-file %s/../../tls/ca.crt" [pwd]]
+            # Note that this loglevel set will effect `Update log level` sentinel test.
             puts $cfg "loglevel debug"
```

Comment:
@yossigo do you remember why you used `debug` here? maybe a leftover from experimentation?

Reply:
@enjoy-binbin i didn't get a response from Yossi, but i imagine this debug config is not needed (otherwise, i ask why we don't run the non-tls tests with debug logging as well, or at least have a comment here).
please try to remove that line and revert the fix in the test.

Reply:
I've thought about removing it, but I didn't bother to try, just one hard code test. The tests passed, this actually much cleaner, thanks.

Reply:
@oranagra I don't remember any specific reason, probably a left over.